src/software/backend/input/network/filter/ball_filter_test.cpp

```diff
@@ -389,15 +390,15 @@ TEST_F(BallFilterTest, ball_moving_fast_in_a_straight_line_with_medium_noise_in_
     double ball_position_variance                = 0.003;
     double time_step_variance                    = 0.001;
     double expected_position_tolerance           = 0.008;
-    Angle expected_velocity_angle_tolernace      = Angle::ofDegrees(2.8);
+    Angle expected_velocity_angle_tolerance      = Angle::ofDegrees(3.0);
```

Comment:
doesn't seem within scope...

Reply:
This test failed when we went from gcc to clang due to floating point error.